src/Raven.Studio/typescript/viewmodels/database/indexes/indexPerformance.ts

```diff
@@ -330,40 +335,58 @@ class metrics extends viewModelBase {
     }
 
     private prepareBrushSection() {
-        const timeRanges = this.extractTimeRanges();
-
-        this.gapFinder = new gapFinder(timeRanges, metrics.minGapSize);
-
-        const collapsedTimeRanges = graphHelper.collapseTimeRanges(timeRanges);
-        //TODO: maybe instead of collaping time range we should graph area chart with # currently indexing as y-axis
+        const timeRanges = this.extractTimeRanges(); 
+        const aggregatedRanges = new rangeAggregator(timeRanges);
+        const workData = aggregatedRanges.aggregate();
+        const maxConcurrentIndexes = aggregatedRanges.maxConcurrentIndexes;
 
         this.brushSection = document.createElement("canvas");
         this.brushSection.width = this.totalWidth;
         this.brushSection.height = metrics.brushSectionHeight;
 
-        const context = this.brushSection.getContext("2d");
-
+        this.gapFinder = new gapFinder(timeRanges, metrics.minGapSize);
         this.xBrushTimeScale = this.gapFinder.createScale(this.totalWidth, 0);
+        
+        this.yBrushValueScale = d3.scale.linear().domain([0, maxConcurrentIndexes]).range([0, metrics.brushSectionIndexesWorkHeight]); 
```

Comment:
reformat to:
d3.scale.linear()
.domain([0, maxConcurrentIndexes])
.range([0, metrics.brushSectionIndexesWorkHeight]);